src/main/kotlin/com/quipalup/katydid/younghuman/SearchYoungHumans.kt

```diff
@@ -0,0 +1,11 @@
+package com.quipalup.katydid.younghuman
+
+import arrow.core.Either
+import javax.inject.Named
+
+@Named
+class SearchYoungHumans(private val youngHumanRepository: YoungHumanRepository) {
+    fun execute(): Either<SearchYoungHumansError, List<YoungHuman>> {
+        return youngHumanRepository.search()
+    }
```

Comment:
In this US, we need to pass some filters in order to get the young humans that are present today